AWSKinesisTests/AWSFirehoseRecorderTests.m

```diff
@@ -191,23 +191,35 @@ - (void)testDiskAgeLimit {
     FirehoseRecorder.diskAgeLimit = 0.0;
 }
 
-- (void)testAll {
-    AWSFirehoseRecorder *firehoseRecorder = [AWSFirehoseRecorder defaultFirehoseRecorder];
-    
-    NSMutableArray *tasks = [NSMutableArray new];
-    for (int32_t i = 0; i < 1234; i++) {
-        [tasks addObject:[firehoseRecorder saveRecord:[[NSString stringWithFormat:@"TestString-%02d\n", i] dataUsingEncoding:NSUTF8StringEncoding]
-                                           streamName:AWSFirehoseRecorderTestStream]];
-    }
-    
-    [[[[AWSTask taskForCompletionOfAllTasks:tasks] continueWithSuccessBlock:^id(AWSTask *task) {
-        sleep(10);
-        return [firehoseRecorder submitAllRecords];
-    }] continueWithBlock:^id(AWSTask *task) {
-        XCTAssertNil(task.error);
```

Comment:
I'm testing `saveRecord` in the below test, and because the default firehose recorder has an invalid pool id, this correctly throws an error.